### ISSUE TITLE
Can't warp to Kamihr Drifts at all

### ISSUE THREAD

**Issue by joepoisonthewell** [AkadenTK/superwarp#20]
Trying to warp to Kamihr Drifts via waypoint only results in "Search returned no matches".

**Comment by AkadenTK** [comment]
IIRC, this is a known issue already. But just to be sure, can you post the logs after running `//sw debug` and then trying again?

**Comment by joepoisonthewell** [comment]
![image](https://user-images.githubusercontent.com/66372606/117488076-d1497a80-af20-11eb-903d-b990a4452f25.png)

**Comment by AkadenTK** [comment]
Are you using autotranslate?

**Comment by joepoisonthewell** [comment]
Yeah, for the hp and the wp. Have also tried single, and with a group with
the all function. I can't remember if I tried without auto translate or
not.

**Comment by AkadenTK** [comment]
Yeah, last I checked the issue was with the autotranslate code stripping through windower. It returns nothing like the text part, but it only happens with kamihr.

**Comment by joepoisonthewell** [comment]
Just tried it manually, works fine. Sorry about the duplicate issue, I
didn't see the previous one. Thanks for your time!

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by AkadenTK** [comment]
No worries.
